cumulus/scripts/migrate_storage_to_genesis/identityMigration.js

```diff
@@ -0,0 +1,108 @@
+const { u8aToHex, hexToString } = require('@polkadot/util');
+const { xxhashAsHex } = require('@polkadot/util-crypto');
+
+const migrateIdentity = async (key, data, api) => {
+	// Register the new `Registration` format type
+	api.registry.register({
+		IdentityInfoNew: {
+		_fallback: 'IdentityInfoTo198',
+		display: 'Data',
+		legal: 'Data',
+		web: 'Data',
+		matrix: 'Data',
+		email: 'Data',
+		pgpFingerprint: 'Option<H160>',
+		image: 'Data',
+		twitter: 'Data',
+		github: 'Data',
+		discord: 'Data',
+		},
+		RegistrationNew: {
+		_fallback: 'RegistrationTo198',
+		judgements: 'Vec<RegistrationJudgement>',
+		deposit: 'Balance',
+		info: 'IdentityInfoNew'
+		},
+		Username: 'Vec<u8>',
+	});
+
+	// We want to migrate `IdentityOf` storage item
+	let IdentityOfkeyToMigrate = "IdentityOf";
```

Comment:
```suggestion
	let IdentityOfKeyToMigrate = "IdentityOf";
```
And in a few other places